src/ngraph/ngraph_sgcompiler.cc

```diff
@@ -236,29 +229,42 @@ void SGCompiler::CompileSubgraph(std::shared_ptr<Graph> sub_graph) {
   CompileNodes(sub_graph->nodes_.back(), sub_graph);
 
   auto f = MakeForwardFunction(sub_graph);
+  if (ngraph_log_graph) {
+    gdumper.dump(f, "_fprop_pre_opt");
```

Comment:
increment?

Reply:
Nope.  IIUC, the increment is _meant_ to correspond compilations of ngraph graphs, so that the serial numbers line up right.

I'm not certain if that discipline is maintained in the other callsites for `dump(...)`, but I know for sure that the callsites I'm adding don't correspond to any ngraph graph compilations.